apps/system/utils/utils_tzselect.c

```diff
@@ -120,7 +120,7 @@ int utils_tzselect(int argc, char **args)
 	strncpy(path, TZDIR, MAX_PATH_LEN);
 	ret = search_tz(path);
 	if (ret) {
-		setenv("TZ", path, 0);
+		setenv("TZ", path, 1);
```

Comment:
How about using definition to know what '1' mean?
```c
#define TZ_ENV_MULTIPLE_SELECTION 1

setenv("TZ", path, TZ_xxxx);
```

cf. please change the definition name with better naming.

Reply:
done